Topic_2/recentFileListBehaviour/src/test/java/com/mycompany/app/recentFileListBehaviour/RecentFileListTest.java

```diff
@@ -0,0 +1,62 @@
+package com.mycompany.app.recentFileListBehaviour;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class RecentFileListTest {
+	private RecentFileList recentFileList;
+	private File file;
+	
+	@Before
+	public void setUp() {
+		recentFileList = new RecentFileList();	
+	}
+	
+	@Test
+	public void whenProgramIsRunForFirstTimeThenListIsEmpty() {
+		assertEquals(0,recentFileList.getList().size());
```

Comment:
It's better to have a method to know if list is empty. This method should return a boolean value.